src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -285,6 +304,11 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
                                 // since the component already exists, the ngAfterInit method of the component will not be called so we must assign the value here manually
                                 this.selectResourceClassComponent.form.controls.resources.setValue(this.resourceClasses[0].id);
                             }
+
+                            // notifies the user that the select ontology has got 0 resource class yet defined.
+                            if ((!this.selectResourceClassComponent || this.selectOntologyComponent.form.controls.ontologies.valueChanges) && this.resourceClasses.length === 0) {
+                                this.errorMessage = 'No resource defined for the select ontology.';
```

Comment:
'No resources defined for the selected ontology.'

Reply:
fixed in 3717fc5